packages/trader/src/AppV2/Components/TradeParameters/Duration/__tests__/duration_container.spec.tsx

```diff
@@ -60,14 +88,26 @@ describe('DurationActionSheetContainer', () => {
         mocked_store: TCoreStores,
         unit = 'm',
         setUnit = jest.fn(),
-        hours = {
-            selected_hour: [0, 0],
-            setSelectedHour: jest.fn(),
-        }
+        selected_hour = [0, 0],
+        setSelectedHour = jest.fn(),
+        end_date = new Date(),
```

Comment:
As I recently broke production tests with Date, so I should warn agains using new Data

Reply:
I think should be okay because in actual case as well I am initializing the end_date state with new Date(). So I don't think it may affect .